dex-it/src/test/scala/com/wavesplatform/it/api/SyncMatcherHttpApi.scala

```diff
@@ -87,8 +87,11 @@ object SyncMatcherHttpApi extends Assertions {
     def placeOrder(order: Order): MatcherResponse =
       sync(async(m).placeOrder(order))
 
-    def placeMarketOrder(order: Order): MatcherResponse =
+    def placeMarketOrder(order: Order): MatcherResponse = {
+      println(
```

Comment:
Do you really need this? Looks like ad-hoc logging
We don't like these side-effects

Reply:
it is useful for debugging tests, ok, I will remove this code and will think about a better solution